WordPress/Classes/Services/BlogService.m

```diff
@@ -560,7 +560,7 @@ - (void)mergeBlogs:(NSArray<RemoteBlog *> *)blogs withAccount:(WPAccount *)accou
     [[ContextManager sharedInstance] saveContext:self.managedObjectContext];
 
     // Ensure that the account has a default blog defined (if there is one).
-    AccountService *service = [[AccountService alloc]initWithManagedObjectContext:self.managedObjectContext];
+    AccountService *service = [[AccountService alloc] initWithCoreDataStack:[ContextManager sharedInstance]];
```

Comment:
Does the BlogService have its own copy of the stack at this point? Could it pass that instead?

Reply:
Not yet. I was going to do the same refactor on `BlogService` the other day, but realized it's not easy—there are more than 1000 lines of code which is double the size of `AccountService`. I think I'm going to leave it at a later stage.